tests/db_migrator_test.py

```diff
@@ -509,6 +509,25 @@ def test_warm_upgrade_to_2_0_2(self):
             diff = DeepDiff(resulting_table, expected_table, ignore_order=True)
             assert not diff
 
+    def test_warm_upgrade__without_mg_to_2_0_2(self):
+        dbconnector.dedicated_dbs['CONFIG_DB'] = os.path.join(mock_db_path, 'config_db', 'cross_branch_upgrade_to_version_2_0_2_input')
+        import db_migrator
+        dbmgtr = db_migrator.DBMigrator(None)
+        # set minigraph_data to None to mimic the missing minigraph.xml scenario
+        dbmgtr.minigraph_data = None
+        dbmgtr.migrate()
+        dbconnector.dedicated_dbs['CONFIG_DB'] = os.path.join(mock_db_path, 'config_db', 'cross_branch_upgrade_without_mg_2_0_2_expected.json')
+        expected_db = Db()
+
+        expected_db
```

Comment:
>expected_db [](http://example.com/codeflow?start=8&length=11)

remove this line?

Reply:
Removed here, and also in the old testcase.